packages/expo/ios/Swift.h

```diff
@@ -3,6 +3,8 @@
 // The generated swift header may depend on some Objective-C declarations,
 // adding dependency imports here to prevent declarations not found errors.
 #import <React/RCTHTTPRequestHandler.h>
+#import <ExpoModulesCore/EXReactNativeFactoryDelegate.h>
```

Comment:
Why do we need to manually include this header? Shouldn't this be automatically imported through `ExpoModulesCore-umbrella.h`?

Reply:
moved the import https://github.com/expo/expo/pull/35050/commits/698d1fef354074db9b5f24f839c47b765c9dfbb2